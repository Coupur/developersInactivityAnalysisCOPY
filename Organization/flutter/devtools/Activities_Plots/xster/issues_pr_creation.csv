id,date,creator_login
426177241,2019-03-27 20:50:01,xster
