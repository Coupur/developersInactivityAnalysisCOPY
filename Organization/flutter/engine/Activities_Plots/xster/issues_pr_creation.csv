id,date,creator_login
206363702,2017-02-08 23:59:43,xster
206377907,2017-02-09 01:28:50,xster
206720299,2017-02-10 06:37:23,xster
206905826,2017-02-10 21:02:38,xster
206921942,2017-02-10 22:23:09,xster
207634802,2017-02-14 20:59:31,xster
209936016,2017-02-24 01:36:39,xster
221712806,2017-04-14 00:08:48,xster
222890823,2017-04-19 22:27:26,xster
230888166,2017-05-24 00:39:27,xster
244442478,2017-07-20 17:40:46,xster
244829991,2017-07-22 07:13:18,xster
245573375,2017-07-26 00:36:06,xster
246499166,2017-07-29 04:50:37,xster
247223202,2017-08-01 22:41:14,xster
248151598,2017-08-05 02:19:21,xster
250208016,2017-08-15 02:48:48,xster
250476848,2017-08-16 00:41:40,xster
257516399,2017-09-13 20:48:29,xster
267143481,2017-10-20 11:17:24,xster
269570929,2017-10-30 11:47:08,xster
274273299,2017-11-15 19:19:43,xster
275968268,2017-11-22 07:26:11,xster
277609439,2017-11-29 02:36:34,xster
299533064,2018-02-22 22:32:49,xster
299535541,2018-02-22 22:42:04,xster
309552757,2018-03-28 22:19:29,xster
315282463,2018-04-18 00:00:17,xster
318539652,2018-04-27 20:27:38,xster
427961835,2019-04-01 23:54:52,xster
456693549,2019-06-16 23:29:30,xster
467274489,2019-07-12 07:56:41,xster
