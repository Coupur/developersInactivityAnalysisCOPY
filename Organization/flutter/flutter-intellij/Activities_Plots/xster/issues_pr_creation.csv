id,date,creator_login
211247188,2017-03-01 23:56:14,xster
230115562,2017-05-20 00:03:37,xster
234688921,2017-06-09 00:08:07,xster
234689736,2017-06-09 00:14:37,xster
351053612,2018-08-16 04:44:25,xster
351781363,2018-08-18 02:04:24,xster
352285813,2018-08-20 20:32:59,xster
353613263,2018-08-24 01:45:05,xster
354933662,2018-08-28 22:55:02,xster
365089269,2018-09-29 02:48:37,xster
368356026,2018-10-09 19:13:10,xster
418917448,2019-03-08 19:06:35,xster
452679443,2019-06-05 19:35:40,xster
