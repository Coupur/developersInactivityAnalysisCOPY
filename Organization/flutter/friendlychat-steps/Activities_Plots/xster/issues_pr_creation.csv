id,date,creator_login
268600827,2017-10-26 01:28:34,xster
