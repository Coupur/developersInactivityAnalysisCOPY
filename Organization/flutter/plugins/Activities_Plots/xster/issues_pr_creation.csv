id,date,creator_login
226714201,2017-05-05 23:14:15,xster
227538787,2017-05-10 00:55:12,xster
241103852,2017-07-06 22:25:25,xster
266469205,2017-10-18 12:09:28,xster
268595057,2017-10-26 00:48:12,xster
286135961,2018-01-04 22:41:54,xster
287223810,2018-01-09 20:42:10,xster
307055391,2018-03-20 22:03:37,xster
309872829,2018-03-29 19:18:05,xster
311822946,2018-04-06 01:39:42,xster
389589029,2018-12-11 04:09:12,xster
