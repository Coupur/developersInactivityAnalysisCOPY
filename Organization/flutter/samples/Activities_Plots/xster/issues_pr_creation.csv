id,date,creator_login
408358676,2019-02-08 23:08:33,xster
430644688,2019-04-08 20:31:34,xster
430665322,2019-04-08 21:23:44,xster
430711643,2019-04-09 00:16:58,xster
435047605,2019-04-19 04:44:00,xster
