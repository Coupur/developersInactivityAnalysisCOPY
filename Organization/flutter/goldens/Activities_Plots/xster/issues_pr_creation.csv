id,date,creator_login
412212270,2019-02-20 02:17:46,xster
436440109,2019-04-24 00:04:39,xster
436961319,2019-04-25 00:32:35,xster
